apps/docs/src/guide/cli/npm-create-fuels/index.md

```diff
@@ -0,0 +1,113 @@
+# npm create fuels
```

Comment:
Looks like [CI](https://github.com/FuelLabs/fuels-ts/actions/runs/8615957986/job/23628037010?pr=2030#step:6:1173) doesn't allow nested folders inside guide, what's the rationale behind moving the file here from `npm-create-fuels.md`?

Reply:
@maschad interesting. I thought it would be a good idea to create a separate folder for `npm create fuels` since with this PR, we added a new child page for it.

Reply:
Understood, seems like the script [can't find it now](https://github.com/FuelLabs/fuels-ts/actions/runs/8631930277/job/23661389040?pr=2030#step:6:1173) as a result though 😩

Reply:
That's so strange, because I actually re-arranged the files now 🤔 @petertonysmith94 could you have a look and tell me if I'm missing something? You seem to be the new docs wizard 😄